integration/kubernetes/helm-chart/alluxio/CHANGELOG.md

```diff
@@ -190,10 +190,13 @@
 - Fix Alluxio CSI `nodeplugin.yaml` indentation
 
 0.6.25
-- Fix Alluxio CSI `nodeplugin.yaml` indentation, add support for dns plocy & change CSI log level
+- Fix Alluxio CSI `nodeplugin.yaml` indentation, add support for dns policy & change CSI log level
 
 0.6.26
 - Add livenessProbe and readinessProbe to values.yaml to allow for overriding
 
 0.6.27
 - Enable Fuse process embedded in worker process
+
+0.6.28
+- Fix Alluxio CSI docker image reference. Move CSI REAEME.md instructions here
```

Comment:
here may not be true as time go by. changelog should be always true no matter what has changed.
use a concrete place to replace the `here`

Reply:
Makes sense. I was a little hesitant to use it as well.